implementations/rust/ockam/ockam_command/src/enroll/command.rs

```diff
@@ -50,21 +60,62 @@ impl EnrollCommand {
         initialize_identity_if_default(&opts, &self.identity);
         node_rpc(rpc, (opts, self));
     }
+
+    pub fn is_running() -> bool {
```

Comment:
Similarly, this method should receive the `CliState` instance as an argument, so you can get its directory as `state.dir`